contrib/onetbb/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "onetbb"
+pkgver = "2021.10.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DTBB4PY_BUILD=OFF",
+    "-DTBB_STRICT=OFF",
+    "-DTBB_TEST=OFF",
+]
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf",
+]
+makedepends = [
+    "libucontext-devel",
+    "linux-headers",
+]
+pkgdesc = "OneAPI Threading Building Blocks"
+maintainer = "psykose <alice@ayaya.dev>"
+license = "GPL-2.0-only"
+url = "https://oneapi-src.github.io/oneTBB"
+source = (
+    f"https://github.com/oneapi-src/oneTBB/archive/refs/tags/v{pkgver}.tar.gz"
+)
+sha256 = "487023a955e5a3cc6d3a0d5f89179f9b6c0ae7222613a7185b0227ba0c83700b"
+# vis breaks symbols
+hardening = []
+# a lot of the tests can deadlock and it's a pain to figure out which to disable
+options = ["!check"]
+
+
+@subpackage("onetbb-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+@subpackage("libtbb")
```

Comment:
is there any reason to split these

Reply:
nothing generally ever links the others and it's 4 line of code

Reply:
it leaves the main package literally empty and these packages have 320k and 128k installed size respectively

just drop the subpackages